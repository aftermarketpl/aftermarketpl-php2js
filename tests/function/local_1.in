$v = 2;
function f_local1($param)
{
    $v = $param;
}
f_local1(3);
return $v;
