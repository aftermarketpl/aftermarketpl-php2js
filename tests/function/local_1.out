var v;
v = 2;
function f_local1(param)
{
    var v;
    v = param;
}
f_local1(3);
return v;
